apps/webapp/src/fetchers/balances/index.ts

```diff
@@ -35,8 +35,8 @@ const getBalances = ({ accountFilter, assetIdFilter }: BalancesProps = {}) => {
   return streamToPromise(iterable);
 };
 
-export const getAssetBalances = async (): Promise<AssetBalance[]> => {
-  const balances = await getBalances();
+export const getAssetBalances = async (balanceProps?: BalancesProps): Promise<AssetBalance[]> => {
```

Comment:
`getAssetBalances` now takes optional arguments so that we can filter by address index or asset ID if desired